src/main/java/edu/usf/cutr/grha/io/CsvParser.java

```diff
@@ -15,28 +15,35 @@
  */
 package edu.usf.cutr.grha.io;
 
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.Reader;
+import java.io.*;
 import java.nio.charset.StandardCharsets;
-import java.util.List;
 
-public abstract class CsvParser {
+public class CsvParser {
```

Comment:
With these changes it doesn't make sense to me to have a common base class that is extended to implement the other parser classes. The methods here are simple single line methods that could be implemented directly in the other classes, either the parsers or the main method or a utility class (in the main class you're currently treating these as if they were utility methods) - let's get rid of this class.